examples/unet_segmentation_3d_dict.py

```diff
@@ -52,15 +52,22 @@
 
 images = sorted(glob(os.path.join(tempdir, 'im*.nii.gz')))
 segs = sorted(glob(os.path.join(tempdir, 'seg*.nii.gz')))
+train_data = list()
+for img, seg in zip(images[:30], segs[:30]):
+    train_data.append({'image': img, 'seg': seg})
```

Comment:
Hi @wyli ,

I think we ask the input data to be a dict and no need to be file names.
Users can pass data to the Dataset directly instead of loading from file.
Thanks.

Reply:
I agree with Wenqi. I think user should just be able to pass image file names and label file names. This extra verbosity should be handled by us. Nobody has their data in this Dict form. Why enforce this when we can handle it ourselves.

Reply:
Updated in the latest patch.
Changed to train_files, val_files
Thanks.